ci/README.md

```diff
@@ -0,0 +1,110 @@
+# Continuous Integration Setup
+
+The CI environment is maintained by the SAP Business Technology Platform on Cloud Foundry team for Routing and Networking (CFN).
+
+All secrets used in the pipeline and surrounding tools are kept in the team's Vault. Those secrets are only necessary for [modifying the pipeline](#updating-or-modifying-the-pipeline).
+## Contact
+
+Credentials and admin access lie with that team. If you have questions or issues, please reach out via the [#haproxy-boshrelease](https://cloudfoundry.slack.com/archives/C0XRT9L22) channel on Slack.
+
+## Setup
+
+Concourse is used as CI system. There are two main types of tests and various release specific steps, all of which are defined in [pipeline.yml](pipeline.yml).
+
+* `unit-tests` runs a series of unit tests on the Ruby templating based configuration file generators and includes linters for all code and test code.
+* `acceptance-tests` and `acceptance-tests-pr` runs a series of acceptance tests developed in Go.
+  * `acceptance-tests-pr` is executed for each PR that is marked with the `run-ci` label, while
+  * `acceptance-tests` is run on new commits to `master`, e.g. after a PR has been merged.
+
+All tests run in Docker. The image `iacbox.common.repositories.cloud.sap/haproxy-boshrelease-testflight` is a built and cached version of building [`Dockerfile`](Dockerfile).
+
+***Note, August 2022***: The image used for acceptance tests is currently outdated and will be updated to a recent state in the near future. An updated version is so far only used for the autobump scripts.
+
+### Unit Tests
+
+Unit tests are executed via `rake` and are contained in [spec/haproxy/templates](../spec/haproxy/templates).
+
+### Acceptance Tests
+
+The acceptance tests run a full BOSH director and exercise creating and running the candidate `haproxy-boshrelease` against a test suite that covers a wide range of features and use cases supported by it.
+
+The code can be found in [acceptance-tests](../acceptance-tests/).
+
+The `haproxy-boshrelease` is deployed via the manifest defined in [manifests/haproxy.yml](../manifests/haproxy.yml). Most of the tests use BOSH ops-files to modify this manifest before running it.
+
+The deployed HAProxy will in most cases have a functioning backend that simply responds with `Hello cloud foundry` to HTTP requests. This backend can be omitted and will lead to a failure state, as HAProxy in its current configuration requires the backend to start.
+
+#### Writing new Acceptance Tests
+
+There are examples for various types of tests already in the source code. Those include startup and draining behaviour, various types of requests and specific configurations where HAProxy modifies the request as well as general functionality checks to avoid regressoins.
```

Comment:
regressoins->regressions
```suggestion
There are examples for various types of tests already in the source code. Those include startup and draining behaviour, various types of requests and specific configurations where HAProxy modifies the request as well as general functionality checks to avoid regressions.
```